data/101/752/473/101752473-alt-quattroshapes_pg.geojson

```diff
@@ -0,0 +1,18 @@
+{
+  "id": 101752473,
+  "type": "Feature",
+  "properties": {
+    "src:alt_label":"quattroshapes_pg",
+    "src:geom":"quattroshapes_pg",
+    "wof:geomhash":"965592fb916a298c53772592b7868a1b",
+    "wof:id":101752473,
+    "wof:repo":"whosonfirst-data-admin-uk"
```

Comment:
Same deal as with other PR: update cc to `ua`

Reply:
fixed in c7ff07a, the rest fixed in dc9d3f3.